pocs/dome/__init__.py

```diff
@@ -30,10 +26,10 @@ def create_dome_from_config(config):
     else:
         brand = dome_config.get('brand')
         driver = dome_config['driver']
```

Comment:
This already made it's way in, but this will fail if `dome` is in config but `driver` is not.

Reply:
Perhaps that was intended but if so we should catch and warn.

Reply:
Good point. This is certainly a fatal error in terms of creating the dome instance, and will most likely keep POCS from working as intended. Probably we should just raise an exception.